examples/llava/clip.cpp

```diff
@@ -1965,6 +1966,16 @@ bool clip_image_preprocess(struct clip_ctx * ctx, const clip_image_u8 * img, cli
                 clip_image_f32_free(res);
             }
         }
+        for (size_t i = 0; i < imgs.size(); ++i) {
+            for (size_t j = 0; j < imgs[i].size(); ++j) {
+                if (imgs[i][j] != nullptr) {
+                    imgs[i][j]->buf.clear();
```

Comment:
Same here, whenever `delete imgs[i][j]` is call, the memory held by `std::vector` will be freed automatically

Reply:
Cool, revised, nice to get advice.